.circleci/config.yml

```diff
@@ -906,4 +906,5 @@ workflows:
             - master
         requires:
         - build_image
+    - operator_upgrade_tests
```

Comment:
Same comment that I put in a previous commit.
I think, after @ivanstanev has split the config.yml in different files, we can't change it directly. Instead you should make changes in the files on the folder .circleci/config and run this commands here to re-generate the config.yml.

Reply:
👍